src/preview/rewriteStyleSheet.ts

```diff
@@ -78,8 +79,8 @@ export const rewriteStyleSheet = (
         sheet.insertRule(newRule, index)
         if (shadowRoot && shadowHosts) shadowHosts.add(shadowRoot.host)
       }
-      if (index > 1000) {
-        warnOnce("Reached maximum of 1000 pseudo selectors per sheet, skipping the rest.")
+      if (index >= limit - 1) {
```

Comment:
Previous config would fail for 1001 selectors, new one fails for 999 with default config. We could keep the same check as we had previously, just use the parametrized limit.
```suggestion
      if (index > limit) {
```